### ISSUE TITLE
NaN output while infering/decoding

### ISSUE THREAD

**Issue by supreet21** [TensorSpeech/TensorFlowTTS#717]
I trained a Tactoron2 model using custom data, 
while the training is going fine as the losses are converging also the predictions from validation set while training also shows, the mel spectograms.
but when I use the same model in the decoding script the mels in .npy files have NaN value. 
Can someone explain why this could be the case?

**Comment by dathudeptrai** [comment]
Can you check the inference and decoding on validation data as well ?
